### DISCUSSION TITLE
模板渲染无法获取数据d.field的问题(只能获得{{d}})

### DISCUSSION THREAD

**Discussion by Kelvinyhl** [layui/layui#1862] [问答]
![1](https://github.com/layui/layui/assets/111147802/53478649-9e3c-4a62-b231-9d2cc73c3f4e)

**Comment by Kelvinyhl** [comment]
当我使用{{d}}的时候，是可以显示数据的（如图）
但是当我使用{{d.id}}的时候，就无法获得字段的数据了！

<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>Quick Start - Layui</title>
  <link href="https://cdnjs.cloudflare.com/ajax/libs/layui/2.9.8/css/layui.css" rel="stylesheet">
</head>
<body>
  

  
  <div class="layui-container">
    
    <div class="layui-row" style="margin-top: 50px;">
      <div class="layui-col-md12">
        <div class="layui-btn-group">
          <button type="button" class="layui-btn layui-btn-xs" id="R1">R1</button>
          <button type="button" class="layui-btn layui-btn-xs" id="R2">R2</button>
          <button type="button" class="layui-btn layui-btn-xs" id="R3">R3</button>
          <button type="button" class="layui-btn layui-btn-xs" id="R4">R4</button>
          <button type="button" class="layui-btn layui-btn-xs" id="R5">R5</button>
          <button type="button" class="layui-btn layui-btn-xs" id="R6">R6</button>
          <button type="button" class="layui-btn layui-btn-xs" id="R7">R7</button>
          <button type="button" class="layui-btn layui-btn-xs" id="R8">R8</button>
          <button type="button" class="layui-btn layui-btn-xs" id="R9">R9</button>
          <button type="button" class="layui-btn layui-btn-xs" id="R10">R10</button>
          <button type="button" class="layui-btn layui-btn-xs" id="R11">R11</button>
          
        </div>
      </div>
    </div>

    <div class="layui-row" style="margin-top: 30px;">
      
      <div class="layui-col-md12">
        <table id="demo" lay-filter="test"></table>
        <script type="text/html" id="typeTpl">
          <div>{{ d }}</div>
        </script>
        
      </div>
S
      
      
    </div>
  </div>
  
  
  
  <script src="https://cdnjs.cloudflare.com/ajax/libs/layui/2.9.8/layui.js"></script> 
  <script>
    layui.use(['table','laytpl'], function(){
      var table = layui.table;
      var form = layui.form;
      var laytpl = layui.laytpl;
      
      // 表格渲染
      table.render({
        elem: '#demo'  // 指定表格容器的id
        ,url: '/df_data' // 数据接口地址

        ,cols: [  // 标题栏
          [ // 表头
            // 请根据实际情况添加列定义，例如：
            {field: 'race_day_race_no_int', title: '场次'},
          
            { title: '用户',width:800, templet: '#typeTpl'},
            
            // 其他列...
          ]
        ]
        ,page: false  // 开启分页
      
        // 其他配置...
      });
    });

    

  </script>
  
</body>
</html>

**Comment by LSL1618** [comment]
模板语法是这样的吧 `{{= d.id }}`
